nixos/modules/services/mail/mailman.nix

```diff
@@ -106,6 +101,14 @@ in {
         '';
       };
 
+      withPostgresql = mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
default should be true
you'd only want to disable this in the rare case, you run psql on another machine